theme/static/js/resourceLandingAjax.js

```diff
@@ -535,6 +548,39 @@ function update_netcdf_file_ajax_submit() {
         }
     });
 }
+function updateModelInstanceMetaSchema() {
+    var $alert_success = '<div class="alert alert-success" id="success-alert"> \
+        <button type="button" class="close" data-dismiss="alert">x</button> \
+        <strong>Success! </strong> \
+        Schema update was successful.\
+    </div>';
+
+    var url = $("#btn-mi-schema-update").attr("data-schema-update-url");
+    $.ajax({
```

Comment:
Pulling the url from the DOM isn't something I would have done, but we can wait for future discussions around VueJS and possible Resource Landing Page cleanup in 2021 and beyond.

Reply:
Sounds good.